exec/cnex/dictionary.c

```diff
@@ -62,6 +63,9 @@ Dictionary *dictionary_createDictionary(int64_t size)
     }
     d->len = 0;
     d->max = size;
+    if (size == 0) {
```

Comment:
Only if it's zero? What if the caller passes 1? Actually, why does the caller even need to pass a `size` here? Does it make sense to do that?

Reply:
It did to me, at the time, because of the CONSD opcode, we already know the size of the array.  I've now removed all size parameters on dictionaries.